packages/dialtone-css/lib/build/less/components/rich-text-editor.less

```diff
@@ -30,11 +37,28 @@
     blockquote {
       margin-left: 0;
       padding-left: var(--dt-space-400);
-      border-left: var(--dt-size-border-300) solid var(--dt-color-foreground-muted-inverted);
+      border-left: var(--dt-size-border-300) solid var(--dt-color-foreground-secondary-inverted);
     }
   }
 }
 
+.d-rich-text-editor-link-bubble-menu {
+  display: flex;
+  font: var(--dt-typography-body-md);
+  background-color: var(--dt-color-surface-primary);
+  border: 1px solid var(--dt-color-border-default);
+  border-radius: 2.8rem;
+
+  // Left-Border on all buttons except the first one (separators)
+  button + button {
+    border-left: 1px solid var(--dt-color-border-default);
+  }
+
+  button {
+    padding: var(--dt-space-300) var(--dt-space-450);
+  }
+}
+
```

Comment:
I provided feedback with @francotirabasso on this and some other things, but the key thing: 

I say this should be a Popover (or at least emulate Popover style) and just use three straight muted buttons. 

Though of course confirm with Franco before doing so.

The result would be effectively...

```html
<dt-popover class="d-p4">
  <dt-stack direction="row" gap="200">
    <dt-button kind="muted" importance="clear">Edit</dt-button>
    <dt-button kind="muted" importance="clear">Open link</dt-button>
    <dt-button kind="danger" importance="clear">Remove</dt-button>
  </dt-stack>
</dt-popover>
```

https://github.com/user-attachments/assets/1df1940d-fcd6-436d-b998-50502838f23a